kv/kvserver: TestClosedTimestampFrozenAfterSubsumption failed

kv/kvserver.TestClosedTimestampFrozenAfterSubsumption [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20609332?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20609332?buildTab=artifacts#/) on release-25.2 @ [78728f1781e0a87b24172638e5cc47346ac6f5ad](https://github.com/cockroachdb/cockroach/commits/78728f1781e0a87b24172638e5cc47346ac6f5ad):


```
=== RUN   TestClosedTimestampFrozenAfterSubsumption
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/7334302ae8c22802de8334e77d6901c9/logTestClosedTimestampFrozenAfterSubsumption3915185671
    test_log_scope.go:76: use -show-logs to present logs inline
    closed_timestamp_test.go:850: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/7334302ae8c22802de8334e77d6901c9/logTestClosedTimestampFrozenAfterSubsumption3915185671
--- FAIL: TestClosedTimestampFrozenAfterSubsumption (36.75s)
=== RUN   TestClosedTimestampFrozenAfterSubsumption/rhs_lease_transfer_while_subsumed
    closed_timestamp_test.go:836: 
        	Error Trace:	pkg/kv/kvserver/closed_timestamp_test.go:836
        	Error:      	Should be true
        	Test:       	TestClosedTimestampFrozenAfterSubsumption/rhs_lease_transfer_while_subsumed
--- FAIL: TestClosedTimestampFrozenAfterSubsumption/rhs_lease_transfer_while_subsumed (29.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClosedTimestampFrozenAfterSubsumption.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55469

--- a/pkg/kv/kvserver/closed_timestamp_test.go
+++ b/pkg/kv/kvserver/closed_timestamp_test.go
@@ -733,11 +733,16 @@ func TestClosedTimestampFrozenAfterSubsumption(t *testing.T) {
 			// Set up the closed timestamp timing such that, when we block a merge and
 			// transfer the RHS lease, the closed timestamp advances over the LHS
 			// lease but not over the RHS lease.
+			//
+			// NB: We want to ensure that the closed timestamp on the LHS is less than
+			// the lease start time for the RHS after the lease transfer. To ensure
+			// this holds and doesn't flake, we increase the target duration to 10
+			// seconds.
 			tc, _, _ := setupClusterForClosedTSTesting(ctx, t, 5*time.Second, 100*time.Millisecond, clusterArgs, "cttest", "kv")
 			defer tc.Stopper().Stop(ctx)
 			sqlDB := sqlutils.MakeSQLRunner(tc.ServerConn(0))
 			sqlDB.ExecMultiple(t, strings.Split(`
-SET CLUSTER SETTING kv.closed_timestamp.target_duration = '5s';
+SET CLUSTER SETTING kv.closed_timestamp.target_duration = '10s';
 SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '100ms';
 SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval = '100ms';
 SET CLUSTER SETTING kv.closed_timestamp.follower_reads.enabled = true;

